validator/fielderror.go

```diff
@@ -14,11 +14,19 @@ func (q FieldError) String() string {
 	var sb strings.Builder
 
 	sb.WriteString("Validation failed on field '" + q.Err.Field() + "'")
-	sb.WriteString(", condition: " + q.Err.ActualTag())
 
-	// Print condition parameters, e.g. max=191 -> max { 191 }
-	if q.Err.Param() != "" {
-		sb.WriteString(" " + q.Err.Param())
+	switch q.Err.ActualTag() {
```

Comment:
@FilipNikolovski  we have added custom err msgs for all tags can you take a look at the msgs? @Gligor23